stdlib/asyncio/subprocess.pyi

```diff
@@ -39,22 +39,42 @@ class Process:
     def kill(self) -> None: ...
     async def communicate(self, input: Optional[bytes] = ...) -> Tuple[bytes, bytes]: ...
 
-async def create_subprocess_shell(
-    cmd: Union[str, bytes],  # Union used instead of AnyStr due to mypy issue  #1236
-    stdin: Union[int, IO[Any], None] = ...,
-    stdout: Union[int, IO[Any], None] = ...,
-    stderr: Union[int, IO[Any], None] = ...,
-    loop: Optional[events.AbstractEventLoop] = ...,
-    limit: int = ...,
-    **kwds: Any,
-) -> Process: ...
-async def create_subprocess_exec(
-    program: _ExecArg,
-    *args: _ExecArg,
-    stdin: Union[int, IO[Any], None] = ...,
-    stdout: Union[int, IO[Any], None] = ...,
-    stderr: Union[int, IO[Any], None] = ...,
-    loop: Optional[events.AbstractEventLoop] = ...,
-    limit: int = ...,
-    **kwds: Any,
-) -> Process: ...
+if sys.version_info >= (3, 10):
+    async def create_subprocess_shell(
+        cmd: Union[str, bytes],  # Union used instead of AnyStr due to mypy issue  #1236
```

Comment:
This issue was closed in 2016. I'm not really sure what the typevar would be good for though.

Reply:
Working on a PR to fix this now